.github/workflows/pylint.yml

```diff
@@ -0,0 +1,16 @@
+name: Linter
+on: pull_request
+jobs:
+  PyLint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user $(egrep "pylint.*" ./requirements.txt)
+          echo "Pylint version: "`pylint --version | head -1`
+          pylint -r n `find . -name "*.py" -type f | awk -F "/" '{print $2}' | uniq`
```

Comment:
This fails to scan some of the files, such as those inside `datumaro/plugins/openvino_plugin/samples`. I would suggest using `git ls-files -z '*.py' | xargs -0 pylint -r n` instead.

Reply:
Yes, indeed. Thanks. Fixed it locally.